### ISSUE TITLE
Default static homepage to Cover Template

### ISSUE THREAD

**Issue by melchoyce** [WordPress/twentytwenty#587]
When you have a static homepage set, it would be nice for the homepage to default to using the  Cover Template.

**Comment by joyously** [comment]
No, it would not. Maybe in the starter content (if that's possible), but otherwise it's changing a post meta field. Selecting one option should not have side effects of changing other options. _(I say that, but that's what we're doing with colors, huh?)_

**Comment by karmatosed** [comment]
I do like the idea that it could default like this, it feels a smart default that would be expected.

**Comment by joyously** [comment]
To make it a default, it can't be a template.
